src/interactions/keyInteraction.ts

```diff
@@ -11,10 +11,12 @@ export module Interactions {
      */
     private _positionDispatcher: Plottable.Dispatchers.Mouse;
     private _keyDispatcher: Plottable.Dispatchers.Key;
-    private _keyCodeCallbacks: { [keyCode: string]: Utils.CallbackSet<KeyCallback> } = {};
+    private _keydownKeyCodeCallbacks: { [keyCode: string]: Utils.CallbackSet<KeyCallback> } = {};
+    private _keyupKeyCodeCallbacks: { [keyCode: string]: Utils.CallbackSet<KeyCallback> } = {};
```

Comment:
Let's rename these variables to be more relatable to their corresponding endpoints.

Maybe
`_keyPressCallbacks` and
`_keyReleaseCallbacks`